editoast/src/views/infra/auto_fixes.rs

```diff
@@ -105,6 +107,30 @@ fn get_operations_fixing_error(
             obj_type: error.get_type(),
         })]
         .to_vec()),
+        InfraErrorType::MissingBufferStop { endpoint } => {
+            let track_id = error.get_id();
+            let position = if *endpoint == Endpoint::Begin {
+                0.
+            } else {
+                infra_cache
```

Comment:
Will add `get_track_section()` to infra_cache methods as soon as PR https://github.com/osrd-project/osrd/pull/5990 is merged.